stores/model.ts

```
@@ -0,0 +1,61 @@
+export const useModelStore = defineStore('model', () => {
+  const models = ref<AIModel[]>([])
+  const selectedModel = ref<string>('gemini-2.0-pro')
+  const isLoading = ref(false)
+  
+  async function fetchModels() {
+    isLoading.value = true
+    try {
+      const response = await fetch('/api/models')
+      const data = await response.json()
+      
+      if (data && data.models) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)